tests/osautoinst/test_running.pm

```diff
@@ -8,8 +8,21 @@ sub run {
     type_string "clear\n";
 }
 
+sub get_log {
+    my ($cmd, $name) = @_;
+    my $ret = script_run "$cmd | tee $name";
+    upload_logs $name if $ret;
+}
+
 sub post_fail_hook {
-    assert_script_run 'openqa-client jobs';
+    script_run 'openqa-client jobs';
+    save_screenshot;
+    get_log 'journalctl --pager-end --no-tail --no-pager -u apache2 -u openqa-scheduler -u openqa-websockets -u openqa-webui -u openqa-worker@1' => 'openqa_services.log';
```

Comment:
This will not work if someone didn't start the worker using systemctl or not as `openqa-worker@1`. But since this is not the production case I assume it is not a problem.